srtcore/core.h

```diff
@@ -139,6 +138,8 @@ enum SeqPairItems
 
 // Extended SRT Congestion control class - only an incomplete definition required
 class CCryptoControl;
+class CUDTUnited;
+class CUDTSocket;
```

Comment:
Required due to removal of `#include "api.h"` due to header hierarchy inversion.